### PR TITLE
VACMS-14116 Clean up Events comments

### PR DESCRIPTION
**Pull request by randimays**
## Summary
Some cleanup of unnecessary comments and unused code before creating a PR for Events unit and Cypress tests.

## Related issue(s)
- https://github.com/department-of-veterans-affairs/va.gov-cms/issues/14116

### CODE REVIEW

File: src/applications/static-pages/events/helpers/index.js [hunk 1]
```diff
@@ -277,27 +248,19 @@ export const deriveStartsAtUnix = (
...
     `${startDateMonth}/${startDateDay}/${startDateYear}`,
     'MM/DD/YYYY',
   );
 
-  // If startsAt is today, then set it to now + 1 hour.
   if (startsAt.isSame(moment(), 'day')) {
     startsAt = moment().add(1, 'hour');
   }
 
-  // If the startsAt is in the past, we need to increase it by a year (since there are only month/day fields).
```
Comment (randimays) [inline src/applications/static-pages/events/helpers/index.js:296]: The code to execute is commented out, so I'm removing the whole check

File: src/applications/static-pages/events/helpers/index.js [hunk 2]
```diff
@@ -307,46 +270,31 @@ export const deriveEndsAtUnix = (
...
 
-  // If there are endsAt fields provided, use those.
   if (endDateMonth && endDateDay) {
     endsAt = moment(
       `${endDateMonth}/${endDateDay}/${endDateYear}`,
       'MM/DD/YYYY',
     ).endOf('day');
   }
 
-  // If the endsAt is in the past, we need to increase it by a year (since there are only month/day fields).
```
Comment (randimays) [inline src/applications/static-pages/events/helpers/index.js:333]: The code to execute is commented out, so I'm removing the whole check

File: src/applications/static-pages/events/helpers/index.js [hunk 3]
```diff
@@ -307,46 +270,31 @@ export const deriveEndsAtUnix = (
...
-  if (endsAt.isBefore(moment())) {
-    // endsAt.add(1, 'year').unix();
-  }
-
-  // If the endsAt is before the startsAt, we need to increase it by another year (since there are only month/day fields).
```
Comment (randimays) [inline src/applications/static-pages/events/helpers/index.js:338]: The code to execute is commented out, so I'm removing the whole check

**Review by eselkin — APPROVED** [review]
The comments were pretty redundant since the code was pretty self explanatory.

**Pull request merged**